Tests/Foundation/TestJSONSerialization.swift

```diff
@@ -1286,8 +1286,8 @@ extension TestJSONSerialization {
     }
 
     func test_serialize_Decimal() {
-        XCTAssertEqual(try trySerialize([-Decimal.leastNonzeroMagnitude]), "[-0.00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001]")
```

Comment:
For now we have reverted to Darwin's Decimal behavior for the size of the mantissa, since it differed between platforms. We will revisit that after the Swift 6.0 release.